src/Fixers.php

```diff
@@ -13,55 +13,80 @@ class Fixers
      * @var string[]
      */
     public static $valid = array(
-        'alias_functions',
         'align_double_arrow',
         'align_equals',
-        'array_element_no_space_before_comma',
-        'array_element_white_space_after_comma',
-        'blankline_after_open_tag',
+        'binary_operator_spaces',
+        'blank_line_after_namespace',
+        'blank_line_after_opening_tag',
+        'blank_line_before_return',
         'braces',
+        'cast_spaces',
         'class_definition',
-        'concat_without_spaces',
+        'combine_consecutive_unsets',
         'concat_with_spaces',
-        'double_arrow_multiline_whitespaces',
-        'duplicate_semicolon',
+        'concat_without_spaces',
+        'dir_constant',
         'echo_to_print',
         'elseif',
-        'empty_return',
         'encoding',
-        'eof_ending',
         'ereg_to_preg',
-        'extra_empty_lines',
-        'function_call_space',
+        'full_opening_tag',
         'function_declaration',
         'function_typehint_space',
+        'hash_to_slash_comment',
+        'heredoc_to_nowdoc',
         'include',
-        'indentation',
-        'linefeed',
-        'line_after_namespace',
-        'list_commas',
-        'logical_not_operators_with_spaces',
-        'logical_not_operators_with_successor_space',
+        'linebreak_after_opening_tag',
         'long_array_syntax',
+        'lowercase_cast',
         'lowercase_constants',
         'lowercase_keywords',
-        'method_argument_default_value',
         'method_argument_space',
         'method_separation',
-        'multiline_array_trailing_comma',
```

Comment:
Yes, the aliases have never been in the valid section, because they are not valid fixers. They only become valid after you map them through the aliases.